### PR TITLE
Propagate Endtime std::chrono changes out

### PR DESCRIPTION
**Pull request by lrusak**
This is the first follow up to #20732.

This pushes out the use of `std::chrono` to avoid casting when not needed. It also improves the readability (IMO).

The changes are quite straightforward and shouldn't cause any behaviour changes.

### CODE REVIEW

**Comment by lrusak** [comment]
jenkins build this please

File: xbmc/Application.cpp [hunk 1]
```diff
@@ -2610,7 +2610,8 @@ bool CApplication::PlayMedia(CFileItem& item, const std::string &player, int iPl
   }
   else if (item.IsPlayList() || item.IsInternetStream())
   {
-    CGUIDialogCache* dlgCache = new CGUIDialogCache(5000, g_localizeStrings.Get(10214), item.GetLabel());
+    CGUIDialogCache* dlgCache =
+        new CGUIDialogCache(5000ms, g_localizeStrings.Get(10214), item.GetLabel());
```
Comment (garbear) [inline xbmc/Application.cpp:2614]: Better than `5s`?

Comment (lrusak) [reply xbmc/Application.cpp]: Fixed, thanks!

**Review by garbear — APPROVED** [review]

**Comment by lrusak** [comment]
jenkins build this please

**Pull request merged**